great_expectations/datasource/fluent/sql_datasource.py

```diff
@@ -823,7 +823,10 @@ def _default_table_name(cls, table_name: str, values: dict, **kwargs) -> str:
                 "table_name cannot be empty and should default to name if not provided"
             )
 
-        return sqlalchemy.quoted_name(value=validated_table_name, quote=True)
+        if sqlalchemy.quoted_name:
```

Comment:
If sqlalchemy is not in the env, it becomes `None` and this raises an `AttributeError`? I don't remember exactly how our compat stuff works

Reply:
@cdkini This evaluates to `False` -- that is our standard check pattern.